submissions/description/cross_site_scripting_xss/stored/url_based/template.md

```diff
@@ -0,0 +1,30 @@
+# URL-Based Stored Cross-Site Scripting
+
+## Overview of the Vulnerability
+
+Stored Cross-Site Scripting (XSS) is a type of injection attack where malicious JavaScript is injected into a website. When a user visits the affected web page, the Javascript executes within that user’s browser in the context of this domain. Stored XSS that is based inside the URL can be found on this domain which allows an attacker to control code that is executed within a user’s browser.
```

Comment:
I think a clarification on the difference here between what "Stored XSS that is based inside the URL" is needed, as some would confuse this with reflected XSS